proxy/__init__.py

```diff
@@ -94,12 +91,10 @@ def get_blob(self, digest: str):
         resp = self.get(
             url,
             allow_redirects=True,
+            stream=True,
         )
-        size = resp.headers.get("content-length")
-        content_digest = sha256_digest(resp.content)
-        if digest != content_digest:
-            UpstreamRegistryError(f"digest mismatch, expected {digest}, got {content_digest}")
-        return resp.content, size
+        size = resp.headers.get("content-length", -1)
```

Comment:
What's the content-length in this case?

Reply:
It's the size of the blob in bytes

Reply:
Even for a stream?

Reply:
Either the `Content-Length` header is the size of the blob in bytes or it will not be present.
See https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding#directives